### PR TITLE
Refactor siteInfo.js with Aphrodite

### LINKED ISSUE #7949: Refactor siteInfo.js with Aphrodite
## Test plan

https://github.com/brave/browser-laptop/pull/7985#issue-218098011

----

**Describe the issue you encountered:** Refactor siteInfo.js with Aphrodite

- Steps to reproduce:
    1. Click the lock icon on the URL bar
    
- Actual result:
  screenshot 2017-03-29 17 01 06

- Expected result:
  screenshot 2017-03-29 16 57 30

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### PR DESCRIPTION
**Pull request by luixxiul**
Closes #7949

Also modify messageBox.js, which was introduced with #7107

- Introduced globalStyles.spacing.dialogInsideMargin
  As the marginTop of title and marginBottom of buttons on messageBox.js had been set to 0.5em, I removed and added them to padding of flyoutDialog. The vaule was calculated this way: `calc(10px + 0.5rem) = 18px`. Also I applied it to the elements inside messageBox.js

- Aligned the buttons to the right (same as messageBox.js)

- Introduced siteInfo__wrapper__large
  As the buttons on mixed content info dialog are so huge that sometimes they are wrapped, epecially on foreign language like Japanese. This is a temporary workaround to avoid it.

- Moved 'denyRunInsecureContent' button to the right to improve UX

- Updated automated test code

Auditors:

Test Plan:
1. Open https://apple.com
2. Click the lock icon on the URL bar
3. Make sure the title and description are aligned
4. Make sure the button is aligned to the right
5. Open http://apple.com
6. Click the unlock icon on the URL bar
7. Make sure the title and description are aligned
8. Open https://mixed-script.badssl.com
9. Click the lock icon
10. Make sure the buttons are aligned to the right and they are not wrapped
11. Click "Load Unsafe Script"
12. Click the unlock icon on the URL bar
13. Click "Stop Loading Unsafe Script"
14. Visit http://jsbin.com/fiyojusahu/edit?html,output
15. Make sure the elements are aligned well
16. Visit http://jsbin.com/sadunogefu/edit?html,output
17. Make sure the switch next to "Prevent this page from creating additional dialogs" is aligned with the body text

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:

### CODE REVIEW

**Comment by luixxiul** [comment]
screenshot 2017-03-30 15 54 49
screenshot 2017-03-30 15 54 59
screenshot 2017-03-30 15 55 10
screenshot 2017-03-30 15 55 21

File: js/components/siteInfo.js [hunk 1]
```diff
@@ -161,4 +234,46 @@ SiteInfo.propTypes = {
...
+
+  secureIcon__wrapper: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  secureIcon__fa: {
+    position: 'absolute'
+  },
+  secureIcon__extendedValidation: {
+    color: 'green'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: Please change this to HEX

Comment (luixxiul) [inline js/components/siteInfo.js]: Updated: https://github.com/brave/browser-laptop/pull/7985/commits/35d55d544008e2e5f6ac98e25dd29b41ae57558e#diff-ab65dc6e11f418450c681ab9221425b0R251

File: js/components/siteInfo.js [hunk 2]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
-      secureIcon = <li><span className='fa fa-unlock' /><span data-l10n-id='insecureConnection' /></li>
+      secureIcon =
+        <div className={css(styles.secureIcon__wrapper)}>
+          <span className={cx({
+            fa: true,
+            'fa-unlock': true,
+            [css(styles.secureIcon__fa)]: true
+          })} />
+          <span className={css(styles.secureIcon__label)}
+            data-test-id='insecureconnection'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: change to `insecureConnection`

File: js/components/siteInfo.js [hunk 3]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
     let viewCertificateButton = null
+
     // TODO(Anthony): Hide it until muon support linux
     if (!platformUtil.isLinux()) {
       viewCertificateButton =
-        <Button l10nId='viewCertificate' className='primaryButton viewCertificate' onClick={this.onViewCertificate} />
+        <div className={css(styles.flexJustifyEnd, styles.viewCertificateButton__wrapper)}>
+          <Button l10nId='viewCertificate'
+            className='primaryButton'
+            data-test-id='viewCertificate'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: `data-test-id` -> `testId`

File: js/components/siteInfo.js [hunk 4]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
-              <Button l10nId='dismissAllowRunInsecureContent' className='primaryButton dismissAllowRunInsecureContentButton' onClick={this.props.onHide} />
-            </li>
-          </ul>
-        </li>
+        <div className={css(styles.connectionInfo__wrapper)}>
+          <div data-test-id='runInsecureContentWarning' data-l10n-id='runInsecureContentWarning' />
+          <div className={css(styles.flexJustifyEnd, styles.viewCertificateButton__wrapper)}>
+            <Button l10nId='allowRunInsecureContent'
+              className='whiteButton'
+              data-test-id='allowRunInsecureContentButton'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: `data-test-id` -> `testId`

File: js/components/siteInfo.js [hunk 5]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
+              onClick={this.onAllowRunInsecureContent}
+            />
+            <Button l10nId='dismissAllowRunInsecureContent'
+              className='primaryButton'
+              data-test-id='dismissAllowRunInsecureContentButton'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: `data-test-id` -> `testId`

File: js/components/siteInfo.js [hunk 6]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
-              <Button l10nId='dismissDenyRunInsecureContent' className='whiteButton dismissDenyRunInsecureContentButton' onClick={this.props.onHide} />
-            </li>
-          </ul>
-        </li>
+        <div className={css(styles.connectionInfo__wrapper)}>
+          <div data-test-id='denyRunInsecureContentWarning' data-l10n-id='denyRunInsecureContentWarning' />
+          <div className={css(styles.flexJustifyEnd, styles.viewCertificateButton__wrapper)}>
+            <Button l10nId='dismissDenyRunInsecureContent'
+              className='whiteButton'
+              data-test-id='dismissDenyRunInsecureContentButton'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: `data-test-id` -> `testId`

File: js/components/siteInfo.js [hunk 7]
```diff
@@ -67,81 +71,150 @@ class SiteInfo extends ImmutableComponent {
...
+              onClick={this.props.onHide}
+            />
+            <Button l10nId='denyRunInsecureContent'
+              className='primaryButton'
+              data-test-id='denyRunInsecureContentButton'
```
Comment (NejcZdovc) [inline js/components/siteInfo.js]: `data-test-id` -> `testId`

**Comment by NejcZdovc** [comment]
@luixxiul can we merge this, because I need this PR to start working on something new

**Comment by luixxiul** [comment]
OK, I'm going to change the base branch as this does not depend on another PR of mine.

**Comment by luixxiul** [comment]
Done! If LGTM, please ping me, thanks!

**Comment by NejcZdovc** [comment]
@luixxiul LGTM, so you can merge it

**Pull request merged**